### DISCUSSION TITLE
minidexed.ins - Minidexed Instrument Definition File for Cakewalk and Reaper DAWs

### DISCUSSION THREAD

**Discussion by blixabloxa** [probonopd/MiniDexed#703] [Show and tell]
Hi All,

I'm not sure if this has already been done, but I'm sharing it anyway as some of you may find it useful if you use mindexed with a DAW.

I've put together an instrument definition file for mindexed patches based on all the default DX7 ROMs - ROM1-4 (A and B) and VRC101-112 (A and B). This file can be loaded into a track in Cakewalk or Reaper to enable changing of the minidexed patches via the DAW. Of course, you will need to have all the ROMs loaded in your sysex folder for it to all line up. 

The file is text based and fairly self explanatory, so can be modified easily to suit your own set up if it is not like mine.

I did a rough check and it seemed to be correct - but I am not guaranteeing that there are no errors in the patch lists for each ROM.

Note I have had to change the filename to minidexed.txt to allow me to attach it. Change it back to minidexed.ins before you use it.
[Minidexed.txt](https://github.com/user-attachments/files/16643640/Minidexed.txt)

Enzo

**Comment by diyelectromusic** [comment]
That's great!

I had one in my sdemp github for use with MidiOx but it is now woefully out of date.  It would be good to have one in the official repository that includes the performances too.

In mine I also included variants for combined ROMs for cases when MiniDexed was used in "128 PC msg" mode like a 128-voice tone module to allow me to select across several banks with patch changes 1-128.

You can see how I set up the different high-level options here: https://github.com/diyelectromusic/sdemp/blob/main/src/RPi/MiniDexed/DX7ROMS.INS

This allowed me to choose between MiniDexed performances; DX7 ROMs for banks/voices; and what I called DX7 Voices which was a flat-across-banks 128 voice definition.

As I say it would be nice if someone wanted to configure and contribute something that could potentially support a range of options (and is nicely up to date!) to the repository :)

Kevin
